components/LoanTermsDisclosure/LoanTermsDisclosure.tsx

```diff
@@ -110,7 +110,7 @@ function CreatePageTerms({ fields }: Pick<LoanTermsDisclosureProps, 'fields'>) {
       <dd>
         {ethers.utils.formatUnits(originationFee, decimals)} {symbol}
       </dd>
-      <dt>Payment Once Loan is Met</dt>
+      <dt>You&lsquo;ll Receive</dt>
```

Comment:
nit: I think this should be `&apos;`, the single quotes are just used to signify a quote within a quote